cli/tags.py

```diff
@@ -1,9 +1,44 @@
 import click
+from ._utils import add_options
+from rich.console import Console
+from rich.table import Table
+from rich import box
 
+__tags_options = [
+    click.option(
+        "-a",
+        "--all",
+        is_flag=True,
+        help="""
+    List all tags
+    """,
+    ),
+]
 
-@click.group("tags", invoke_without_command=True)
-def tags():
-    """
-    List tags
-    """
-    pass
+
+@click.command(short_help="Get tags")
+@add_options(__tags_options)
+@click.pass_context
+@click.argument("value", required = False)
+def tags(ctx, value, all):
+    if all:
+        error, answer = ctx.obj.get_all_tags()
+
+    else:
+        if value:
+            error, answer = ctx.obj.get_tag_by_id(value)
+    print_table(answer)
+
+
```

Comment:
leave 2 blank lines not 3. Use `black` for formatting.

Reply:
sorry about that. i will run flake and black